admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MapperTest.java

```diff
@@ -137,6 +140,52 @@ public void testSetCreateMetaDataWithIsActive() {
 		assertNotNull(entity.getCreatedBy());
 		assertFalse(entity.getIsActive());
 	}
+	@Test
+	public void mapUpdateHolidaysTest001(){
+		List<Holiday> holidays=new ArrayList<>();
+		Holiday holiday=new Holiday();
+		holiday.setHolidayDate(LocalDate.now());
+		holiday.setHolidayId(123);
+		holiday.setLangCode("eng");
+		holiday.setIsActive(true);
+		holidays.add(holiday);
+
+		List<HolidayUpdateDto> holidayDtos = new ArrayList<>();
+		HolidayUpdateDto holidayUpdateDto=new HolidayUpdateDto();
+		holidayUpdateDto.setHolidayName("name");
+		holidayUpdateDto.setHolidayDesc("desc");
+		holidayUpdateDto.setHolidayId(345);
+		holidayUpdateDto.setHolidayDate(LocalDate.now());
+		holidayUpdateDto.setHolidayId(7);
+		holidayUpdateDto.setLangCode("eng");
+		holidayUpdateDto.setLocationCode("location123");
+		holidayDtos.add(holidayUpdateDto);
+
+		MapperUtils.mapUpdateHolidays(holidays);
```

Comment:
The test case must assert either exception or a result.

Reply:
Added an assertions in the test methods and updated the method names